### ISSUE TITLE
Mising favicon path

### ISSUE THREAD

**Issue by explunit** [IdentityServer/IdentityServer3#2663]
- [x] I read and understood how to enable [logging](https://identityserver.github.io/Documentation/docsv2/configuration/logging.html)

### Question / Issue
Very minor issue: If I'm reading this correctly, [layout.html](https://github.com/IdentityServer/IdentityServer3/blob/master/source/Core/Services/DefaultViewService/PageAssets/layout.html) does not reference the [default favicon in assets](https://github.com/IdentityServer/IdentityServer3/blob/master/source/Core/Services/DefaultViewService/HttpAssets/favicon.ico), and this causes extra 404's. [Welcome.html](https://github.com/IdentityServer/IdentityServer3/blob/master/source/Core/Services/DefaultViewService/PageAssets/welcome.html#L8) does include a favicon reference.

### Relevant parts of the log file

```
n/a
```

**Comment by brockallen** [comment]
we don't do a favicon on the normal pages since it should be your icon, not ours.

**Comment by explunit** [comment]
True, but it becomes a problem of where exactly you specify it, short of manually copying it to the root of the web server. In a path like `/myhostapp/identityserver`, myhostapp has no opportunity to set the favicon. The only time it really works well is when the host app is at the root of the web server.

**Comment by brockallen** [comment]
Ok, I'll see what can be done.

**Comment by brucehunter** [comment]
To get the favicon.ico working just put it in the root path of the WebAPI project. I don't use the custom layouts yet, but I have my favicon.ico working. (IdentityManager however, this doesn't work at all).
